client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -303,6 +300,21 @@ export const useCommandsArrayWpcom = ( {
 			},
 			icon: helpIcon,
 		},
+		{
+			name: 'openSiteDashboard',
+			label: __( 'Open site dashboard' ),
+			searchLabel: [
+				_x( 'open site dashboard', 'Keyword for the Open site dashboard command' ),
+				_x( 'admin', 'Keyword for the Open site dashboard command' ),
+				_x( 'wp-admin', 'Keyword for the Open site dashboard command' ),
+			].join( ' ' ),
+			context: [ '/sites' ],
+			callback: setStateCallback( 'openSiteDashboard', __( 'Select site to open dashboard' ) ),
+			siteFunctions: {
+				onClick: ( param ) => commandNavigation( `/home/${ param.site.slug }` )( param ),
+			},
+			icon: dashboardIcon,
+		},
```

Comment:
Let's keep this command in the same order it was before.

Reply:
https://github.com/Automattic/wp-calypso/pull/87790/commits/142b8f14dd3e140e029f3ad98592be0f7d67b7ac